test/integration/targets/iosxr_banner/tests/cli/basic-motd.yaml

```diff
@@ -20,8 +22,8 @@
 - assert:
     that:
       - "result.changed == true"
-      - "'this is my motd banner' in result.commands[0]"
-      - "'that has a multiline' in result.commands[0]"
+      - "'this is my motd banner' in result.commands[1]"
```

Comment:
This might catch people out, maybe worth adding to `porting_guide_2.5.rst`

Reply:
agree, will add a note.